templates/KnockoutSpa/tsconfig.json

```diff
@@ -3,8 +3,13 @@
     "moduleResolution": "node",
     "target": "es5",
     "sourceMap": true,
-    "skipDefaultLibCheck": true
+    "skipDefaultLibCheck": true,
+    "typeRoots": [ "node_modules/@types" ],
```

Comment:
Removed the `typeRoots` part: (`node_modules/@types`) is default.
The part where `types:[]` is set is needed otherwise it won't make them necessary during compilation.